graphql_api/types/account/account.graphql

```diff
@@ -1,4 +1,5 @@
 type Account {
   name: String!
   oktaConfig: OktaConfig
+  totalSeatCount: Int
```

Comment:
nit: should this be Int!

Reply:
I was copying the `config.seatCount` type, but yes I think you're right, it is always defined

Reply:
will update